src/pages/EmployeeListing/components/EmployeeTable.tsx

```diff
@@ -1,26 +1,41 @@
-import { useEffect, useState } from "react";
-import { Employee } from "../models/employee";
-import { searchEmployees } from "../services/helpers";
+import { Employee } from "../../../models";
 import EmployeeRow from "./EmployeeRow";
 
 export default function EmployeeTable({
```

Comment:
All components must adhere to SOLID and DRY principles as much as possible. 
1. Any functionality which doesn't affect the styling (JSX behaviour) must be moved out of the component
2. The `<th>` wrapped elements seem to be repetitively declared and must instead be created by array iteration.

Reply:
done